daemon/daemon.go

```diff
@@ -244,11 +244,9 @@ func (d *Daemon) AlwaysAllowLocalhost() bool {
 	return d.conf.alwaysAllowLocalhost
 }
 
-func (d *Daemon) PolicyEnabled() bool {
-	return d.conf.Opts.IsEnabled(endpoint.OptionPolicy)
-}
-
 func (d *Daemon) PolicyEnforcement() string {
+	d.conf.EnablePolicyMU.RLock()
+	defer d.conf.EnablePolicyMU.RUnlock()
```

Comment:
`defer d.conf.EnablePolicyMU.RUnlock()` creates overhead for a small function like this, you can write it as 
```
func (d *Daemon) PolicyEnforcement() (pe string) {
     d.conf.EnablePolicyMU.RLock()
     pe = d.conf.EnablePolicy
     d.conf.EnablePolicyMU.RUnlock()
     return
}
```